3rdparty/install-libcxx.sh

```diff
@@ -93,7 +96,7 @@ cd ../..
 #    -DCMAKE_CC_COMPILER="$DSTDIR/usr/bin/musl-gcc"
 cd libcxx
 rm -rf build && mkdir build && cd build
-cmake -DCMAKE_INSTALL_PREFIX="$DSTDIR/usr" \
+cmake3 -DCMAKE_INSTALL_PREFIX="$DSTDIR/usr" \
```

Comment:
This does not work on Ubuntu, but it does when using the `$CMAKE` variable instead :)